spec/features/user_spec.rb

```diff
@@ -13,7 +13,7 @@
         expect(page).to have_content(user.name)
       end
 
-      it 'allows edit cards' do
+      it 'allows edit cards_mailer' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.